documentation/release_notes.rst

```diff
@@ -8,6 +8,117 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
+- Inclusion of ``<oneapi/dpl/dynamic_selection>`` prior to ``<oneapi/dpl/random>`` may result in compilation errors.
+  Include ``<oneapi/dpl/random>`` first as a workaround.
+- Incorrect results may occur when using the ``oneapi::dpl::experimental::philox_engine`` algorithm with no predefined template
+  parameters with `word_size` values other than 64 and 32.
+- Incorrect results or a synchronous SYCL exception may be observed with DPC++ execution policies when compiling the
+  following algorithms with -O0: ``exclusive_scan``, ``inclusive_scan``, ``transform_exclusive_scan``,
+  ``transform_inclusive_scan``, ``copy_if``, ``remove``, ``remove_copy``, ``remove_copy_if``, ``remove_if``,
+  ``partition``, ``partition_copy``, ``stable_partition``, ``unique``, ``unique_copy``, and ``sort``.
+- The value type of the input sequence should be convertible to the type of the initial element for the following
+  algorithms with DPC++ execution policies: ``transform_inclusive_scan``, ``transform_exclusive_scan``,
+  ``inclusive_scan``, and ``exclusive_scan``.
+- The complexity of the following algorithms with DPC++ execution policies may exceed the number of applications of the
+  user-provided predicate or equality operator compared to the C++ standard requirements: ``copy_if``, ``remove``,
+  ``remove_copy``, ``remove_copy_if``, ``remove_if``, ``partition_copy``, ``unique``, and ``unique_copy``. In all cases,
+  the predicate or equality operator is applied ``O(n)`` times.
+- The ``adjacent_find``, ``all_of``, ``any_of``, ``equal``, ``find``, ``find_if``, ``find_end``, ``find_first_of``,
+  ``find_if_not``, ``includes``, ``is_heap``, ``is_heap_until``, ``is_sorted``, ``is_sorted_until``, ``mismatch``,
+  ``none_of``, ``search``, and ``search_n`` algorithms may cause a segmentation fault when used with a DPC++ execution
+  policy for CPU device, and built on Linux with Intel® oneAPI DPC++/C++ Compiler 2025.0.0 and -O0 -g compiler options.
+
+
+
+Existing Issues
+^^^^^^^^^^^^^^^
+See oneDPL Guide for other `restrictions and known limitations`_.
+
+- ``histogram`` algorithm requires the output value type to be an integral type no larger than 4 bytes
+  when used with an FPGA policy.
+- When compiled with ``-fsycl-pstl-offload`` option of Intel oneAPI DPC++/C++ compiler and with
+  libstdc++ version 8 or libc++, ``oneapi::dpl::execution::par_unseq`` offloads
+  standard parallel algorithms to the SYCL device similarly to ``std::execution::par_unseq``
+  in accordance with the ``-fsycl-pstl-offload`` option value.
+- When using the dpl modulefile to initialize the user's environment and compiling with ``-fsycl-pstl-offload``
+  option of Intel® oneAPI DPC++/C++ compiler, a linking issue or program crash may be encountered due to the directory
+  containing libpstloffload.so not being included in the search path. Use the env/vars.sh to configure the working
+  environment to avoid the issue.
```

Comment:
Need to clarify with @rarutyun if this issue still applies.

Reply:
I think the issue should not be applicable anymore. While I am not 100% sure what dpl modulefile is doing, oneDPL doesn't contain libpstloffload.so anymore. If some linking issue exists I believe it's compiler bug because now they hold libpstloffload and its counterpart on Windows.

As far as PSTL Offload concerned there should not be any questions to oneDPL if users take the latest compiler, because if everything is correctly implemented on the compiler side, PSTL offload should work with any oneDPL

Reply:
I agree that it should not be an issue with the PSTL offload .so file relocated to the compiler.  I've removed the issue.